main_test.go

```diff
@@ -90,6 +90,10 @@ func TestEndpoints(t *testing.T) {
 		{"/search?type=input&prerelease=true", "/search", "search-input-packages.json", searchHandler(testLogger, indexer, testCacheTime)},
 		{"/search?type=input&package=integration_input&prerelease=true", "/search", "search-input-integration-package.json", searchHandler(testLogger, indexer, testCacheTime)},
 		{"/search?type=integration&package=integration_input&prerelease=true", "/search", "search-integration-integration-package.json", searchHandler(testLogger, indexer, testCacheTime)},
+		{"/search?capabilities=observability,security&prerelease=true", "/search", "search-prerelease-capabilities-observability-security.json", searchHandler(testLogger, indexer, testCacheTime)},
+		{"/search?capabilities=none&prerelease=true", "/search", "search-prerelease-capabilities-none.json", searchHandler(testLogger, indexer, testCacheTime)},
+		{"/search?spec.min=1.1.0&spec.max=2.10.0&prerelease=true", "/search", "search-spec-min-max-2.10.0.json", searchHandler(testLogger, indexer, testCacheTime)},
+		{"/search?spec.max=3.1.0&prerelease=true", "/search", "search-spec-default-min-max-3.1.0.json", searchHandler(testLogger, indexer, testCacheTime)},
```

Comment:
Added some tests for search endpoint using capabilities and spec parameters.

Some versions of the `example` and `foo` test packages have been updated:
- adding capabilities
- updating `format_version` number